test/TestProjects/FirstTest-Cadl/Generated/cadl.json

```diff
@@ -0,0 +1,400 @@
+{
+ "$id": "1",
+ "Name": "cadlfirsttest",
+ "Description": "This is a sample cadl project.",
+ "ApiVersions": [
+  "0.1.0"
+ ],
+ "Enums": [
+  {
+   "$id": "2",
+   "Name": "SimpleEnum",
+   "Namespace": "CadlFirstTest",
+   "Description": "Simple enum",
+   "EnumValueType": "String",
+   "AllowedValues": [
+    {
+     "$id": "3",
+     "Name": "One",
+     "Value": "1"
+    },
+    {
+     "$id": "4",
+     "Name": "Two",
+     "Value": "2"
+    },
+    {
+     "$id": "5",
+     "Name": "Four",
+     "Value": "4"
+    }
+   ],
+   "IsExtensible": false,
+   "IsNullable": false
+  },
+  {
+   "$id": "6",
+   "Name": "ExtensibleEnum",
+   "Namespace": "CadlFirstTest",
+   "Description": "Extensible enum",
+   "EnumValueType": "String",
+   "AllowedValues": [
+    {
+     "$id": "7",
+     "Name": "One",
+     "Value": "1"
+    },
+    {
+     "$id": "8",
+     "Name": "Two",
+     "Value": "2"
+    },
+    {
+     "$id": "9",
+     "Name": "Four",
+     "Value": "4"
+    }
+   ],
+   "IsExtensible": true,
```

Comment:
Hm... in the `InputEnumType`, the property is called `IsExtendable`. @annelo-msft, what do you think should be a correct name for the property?

Reply:
@chunyu3, please rename `IsExtendable` properties into `IsExtensible` in autorest.csharp source

Reply:
I updated `IsExtendable` to `IsExtensible` in autorest.csharp and fresh the cadl test project.